Cython/Compiler/Symtab.py

```diff
@@ -31,6 +29,167 @@
     '_Bool', '_Complex'', _Imaginary', 'inline', 'restrict',
 }
 
+# Builtins python 3.12.2
+KNOWN_PYTHON_BUILTINS_VERSION = (3, 12, 2, 'final', 0)
```

Comment:
These probably didn't change since 3.12.0. I'd try to use the oldest possible version since which there was no change. Remember that using a later version disables the test for more Python installations.
```suggestion
# Builtins as of Python 3.12.0
KNOWN_PYTHON_BUILTINS_VERSION = (3, 12, 0, 'final', 0)
```